yt/python/yt/wrapper/auth_commands.py

```diff
@@ -57,3 +120,9 @@ def list_user_tokens(user, password=None,
         "list_user_tokens",
         params=params,
         client=client)
+
+
+def list_user_tokens_interactive(user, client=None):
```

Comment:
It is not a strong opinion, but let consider placing all of these `*_interactive` methods to `cli_impl.py`, since interactive commands look relevant for CLI scenarios only

Reply:
Moved to cli_impl.py.